src/sync.js

```diff
@@ -1,14 +1,20 @@
 var debug = require('debug')('feathers-sync');
 var mongo = require('./mongodb');
 var redis = require('./redis');
+var amqp = require('./amqp');
 
 module.exports = function(config) {
 
+  var proto = '';
   config = config || { db: 'mongodb://localhost:27017/sync' };
 
-  var proto = config.db.split('://')[0];
+  if(config.db){
```

Comment:
Maybe we should just change this to `config.uri` since it's no longer just dbs anymore. @daffl?